### PR TITLE
Create overview page

### LINKED ISSUE #2: Create overview over all subjects

### PR DESCRIPTION
**Pull request by axel1200**
Fixes #2 
Well you can try it and see it would take it or rather redo XD

### CODE REVIEW

**Comment by Rackover** [comment]
Not sure about the use of \<table/tr/td\> elements. I avoided them in my layout because they are unreliable, and using them nowadays is discouraged. I would rather see you use CSS flexboxes than tables.

**Comment by Rackover** [comment]
https://github.com/FAForever/faf-voting/pull/3/files#diff-ff39f0647139394adcc865bda9d056ffR259
I deliberately named the button "Back to overview" and not "Vote again", because "Vote again" is misleading : you **can't** vote again on a subject, what you can do is **select another subject** and then vote.
This button doesn't allow to vote again, it allows to vote for another subject. This small change isn't particularly useful and should be reverted, or changed into something clearer.

**Comment by axel1200** [comment]
Lets just delete the button then, it is no longer needed

**Comment by axel1200** [comment]
Do you know what i am doing wrong:
I have the header and the footer in the same colour but when I look at the website they seem to have slightly differnt colours how can that be?

**Comment by micheljung** [comment]
@Rackover err.. what did you deliberately do? ;D https://github.com/FAForever/faf-voting/commit/d1ca0d64947bc35c66b34351c36b30b74dc397c2#diff-ff39f0647139394adcc865bda9d056ffL214

**Comment by Rackover** [comment]
*facepalm* okay, right, I forgot to do it and I thought I had. Sorry for that. 
Still, naming this "vote again" is a bad idea to me.

**Comment by axel1200** [comment]
Do you know what i am doing wrong:
I have the header and the footer in the same colour but when I look at the website they seem to have slightly differnt colours how can that be?

**Comment by axel1200** [comment]
image

**Comment by axel1200** [comment]
@Rackover @micheljung would be lovely if you could have a look again as ladder voting might come soon and I would love to have this done then.

**Comment by axel1200** [comment]
image
image

File: public/overview.html [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+
+            for (let i = 0; i < subjects.length; i++) {
+                let card = document.createElement("div");
+                card.className = "card";
+                receiver.append(card);
+
+                let cardHeader = document.createElement("div");
+                cardHeader.className = "card-header";
+                let subject = subjects[i];
+                cardHeader.innerHTML = subject.attributes.subject;
```
Comment (Rackover) [inline public/overview.html]: Unless subject .attributes.subject contains HTML tags, you can use textContent - it's faster.

Comment (axel1200) [reply public/overview.html]: no it does not

Comment (axel1200) [reply public/overview.html]: Cool

**Comment by axel1200** [comment]
Ofc I tested it

**Pull request merged**